packages/bpk-mixins/src/mixins/_borders.scss

```diff
@@ -264,3 +264,19 @@
 @mixin bpk-border-left-xl($color, $inset: inset) {
   @include _bpk-border-left($bpk-border-size-xl, $color, $inset);
 }
+
+@mixin bpk-datatable-border-left($bpk-border-size-datatable) {
```

Comment:
Hmm. I don't think I'd include them in the package to be exported given they're not meant to be used outside BPK and were just a way to avoid duplicating code in the datatable component 🤔

Reply:
Moved back to the component and made two private mixins public